src/main/java/seedu/address/logic/candidate/EditCandidateCommand.java

```diff
@@ -59,7 +59,10 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Candidate: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This candidate already exists in the address book.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "Candidate with Email:"
+            + " [ %1$s ] already exists in the HR Manager";
+    public static final String MESSAGE_ILLEGAL_PERSON_STATUS = "Candidate: [ %1$s ]"
+            + " already have scheduled interview(s)";
```

Comment:
Maybe say that because of this, you cannot set the status to Applied? Because edit_c can take a bunch of params, so the user maybe confused on what you mean.